tests/models/caffe2Models/concat_predict_net.pbtxt

```diff
@@ -0,0 +1,22 @@
+name: "concatx3"
+op {
+  input: "inputs_0"
+  input: "inputs_1"
+  input: "inputs_2"
+  output: "concat_result"
+  output: "split_info"
+  name: ""
+  type: "Concat"
+  arg {
+    name: "add_axis"
```

Comment:
Have you tried this net in Caffe2? If add_axis=0, it should just sum dimensions.

Reply:
Good point I haven’t!
And I managed to screw the writing of the test.

Reply:
I concur this is what add_axis does. The documentation is really messed up it says it modifies the axis of the inputs!